### PR TITLE
Checkout: add summary features for each of the WP.com plans

### PR DESCRIPTION
**Pull request by michaeldcain**
In #42854 we added plan features to the Checkout summary for the business plan. This adds features for all active WordPress.com plans.

**All Plans:**
- "Free domain for one year" unless a domain is in the cart, then the bundling message will be on the domain line item

**Personal:**
- Remove WordPress.com ads
- Limit your content to paying subscribers

**Premium:**
- Unlimited access to our library of Premium Themes
- Subscriber-only content and simple payment buttons
- Track your stats with Google Analytics

**Business:**
- Install custom plugins and themes
- Drive traffic to your site with our advanced SEO tools
- Track your stats with Google Analytics
- Real-time backups and activity logs

**Ecommerce:**
- Install custom plugins and themes
- Accept payments in 60+ countries
- Integrations with top shipping carriers
- Unlimited products or services for your online store
- eCommerce marketing tools

**To test:**
- add a plan to your cart and visit Checkout with `?flags=composite-checkout-testing`
- verify that the checkout summary has the correct features
- switch plan variation and verify that the features remain
- add a domain and verify that the "Free domain for one year" feature is removed (a separate domain item should be seen)
- repeat for other plans

### CODE REVIEW

File: client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js [hunk 1]
```diff
@@ -169,6 +169,25 @@ function CheckoutSummaryPlanFeatures() {
 
 function getPlanFeatures( plan, translate, hasDomainsInCart ) {
 	if (
+		'personal-bundle' === plan.wpcom_meta?.product_slug ||
+		'personal-bundle-2y' === plan.wpcom_meta?.product_slug
+	) {
+		return [
+			! hasDomainsInCart && translate( 'Free domain for one year' ),
+			translate( 'Remove WordPress.com ads' ),
+			translate( 'Limit your content to paying subscribers.' ),
```
Comment (a8ci18n) [inline client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js:178]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Limit content to paying subscribers.' )` *ES Score: 9*

**Comment by nbloomf** [comment]
Cain you are supposed to be on vacation!

All combinations of plan+domain show the expected lists of features -- nice work.

**Comment by fditrapani** [comment]
> Cain you are supposed to be on vacation!

Seriously! Looks good to me too.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3786479

Hi @michaeldcain, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Limit your content to paying subscribers.`
- `Unlimited access to our library of Premium Themes`
- `Subscriber-only content and simple payment buttons`
- `Real-time backups and activity logs`
- `Unlimited products or services for your online store`
- `eCommerce marketing tools for emails and social networks`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3786479).

**Pull request merged**